### PR TITLE
Example Stub Article

### PR DESCRIPTION
**Pull request by shannonbux**
A suggestion from an interviewee

### CODE REVIEW

**Comment by KyleAMathews** [comment]
Deploy preview failed.

Built with commit ff7ed5d075443bda44ccd9712ea884989b7305cb

https://app.netlify.com/sites/using-glamor/deploys/59e539340b79b76d2e960809

File: docs/docs/coming-from-react-to-gatsby [hunk 1]
```diff
@@ -0,0 +1,9 @@
+---
+title: Coming from React to Gatsby: What You Need to Know
+---
+
+This is a stub. Help our community expand it.
+
+The [Gatsby Style Guide](/docs/gatsby-style-guide) will help ensure your pull request gets accepted.
```
Comment (KyleAMathews) [inline docs/docs/coming-from-react-to-gatsby]: The verb is missing kinda. Perhaps "Please review the Gatsby Style Guide to help ensure your pull request is accepted"?

File: docs/docs/coming-from-react-to-gatsby [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+
+More Information:
```
Comment (KyleAMathews) [inline docs/docs/coming-from-react-to-gatsby]: What are you planning on adding here?

**Comment by shannonbux** [comment]
So, I'm struggling to imagine the user flow with these stub articles and the appropriate kinds of links that would make sense based on the flow. My big questions: 

1. When Bellatrix Lestrange looks at a stub article (on Gatsbyjs.org or Github) and clicks on the link to open the Gatsby Style Guide, should it open in Gatsbyjs.org or Github? Are there reasons to choose one over the other?
2. For the convenience of average Joe, should the stub articles contain any sort of link to their Github counterpart? It seems like that might help if people access the stub articles primarily on Gatsbyjs.org, but it's totally redundant if they are accessing them in Github...

**Comment by KyleAMathews** [comment]
We want users to always browse on gatsbyjs.org

The only link to GitHub is the "edit this page" link which will be on every page derived from markdown files. I'll be adding that today.

**Comment by shannonbux** [comment]
Ok. So the link on Gatsbyjs.org isn't up yet, I'm sure b/c some edits are still happening from my end--I just submitted some edits and will wait for it to appear on Gatsbyjs.org!

**Comment by KyleAMathews** [comment]
Yeah, your edits don't show up on gatsbyjs.org until your Pull Requests are merged. Pull requests are like a temporary scratchpad we can try changes on and then merge them into the "master" version once the changes are ready.

**Comment by KyleAMathews** [comment]
So now that I've merged this, gatsbyjs.org will get rebuilt with the changes!

**Pull request merged**